client/settings/payment-methods-selector/index.js

```diff
@@ -4,25 +4,54 @@
  */
 import { __ } from '@wordpress/i18n';
 import { Button, Modal } from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { useState, useCallback, useEffect } from '@wordpress/element';
 import { HorizontalRule } from '@wordpress/primitives';
 
 /**
  * Internal dependencies
  */
 
-import { addSelectedPaymentMethods } from 'data';
+import { useEnabledPaymentMethodIds } from 'data';
 import PaymentMethodCheckboxes from '../../components/payment-methods-checkboxes';
 import PaymentMethodCheckbox from '../../components/payment-methods-checkboxes/payment-method-checkbox';
 import './style.scss';
 
-const PaymentMethodsSelector = ( { onClose, enabledPaymentMethods = [] } ) => {
-	const [ paymentMethods, setPaymentMethods ] = useState(
-		enabledPaymentMethods.reduce( ( acc, value ) => {
-			acc[ value ] = true;
-			return acc;
-		}, {} )
-	);
+const PaymentMethodsSelector = ( { className } ) => {
+	const availablePaymentMethods = [
```

Comment:
👍 I have a PR that will solve this: https://github.com/Automattic/woocommerce-payments/pull/1848

Reply:
Very nice!